src/test/datascience/notebook/notebookEditor.vscode.test.ts

```diff
@@ -54,8 +55,6 @@ suite('Notebook Editor tests', () => {
 
     test('Run cells below', async function () {
         // add some cells
-        // https://github.com/microsoft/vscode-jupyter/issues/4250
-        this.skip();
         await insertCodeCell('print("0")', { index: 0 });
         await insertCodeCell('print("1")', { index: 1 });
         await insertCodeCell('print("2")', { index: 2 });
```

Comment:
We need to ensure the 3rd cell is selected here, we aren't doing that

Reply:
I'll select the first one. Then it executes below and checks that the third one was executed.

Reply:
Please could you select the second cell. & then add code to test that first cell did not get executed. This would be a better test, else we're just testing whetehr the whole document ran & not just specific cells.